astropy/coordinates/earth.py

```diff
@@ -395,6 +403,33 @@ def get_itrs(self, obstime=None):
                                      for the location of this object at the
                                      default ``obstime``.""")
 
+    def get_gcrs_posvel(self, obstime):
```

Comment:
I just realised there is a `get_itrs` but there the `obstime` is optional and we also have the `itrs` property. Here, a time is obviously needed, so the situation is not quite the same.  Still, given that precedent, probably fine just to keep as is.